include/mbedtls/mbedtls_config.h

```diff
@@ -522,6 +522,23 @@
  */
 //#define MBEDTLS_AES_FEWER_TABLES
 
+/**
+ * \def MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
+ *
+ * Use only 128-bit keys in AES operations to save ROM.
+ *
+ * Uncommenting this macro removes support for AES operations that use 192-
+ * or 256-bit keys.
+ *
+ * Tradeoff: Uncommenting this macro reduces the size of AES code by about 4%.
+ *
+ * Module:  library/aes.c
+ *
+ * Requires: MBEDTLS_AES_C
+ *
```

Comment:
(minor) Generally we don't have an empty ` *` line before the closing `*/` line in this file. The section about (line 521) is unusual in that it does have the extra line, but this line shouldn't be here and line 521 could reasonably be deleted (separate commit)